src/clj_sqs_extended/core.clj

```diff
@@ -70,34 +64,73 @@
     (.deleteBucket s3-client bucket-name)))
 
 (defn create-queue
-  "Creates a new queue with the passed name via the passed sqs client interface."
   ([sqs-client]
-   (create-queue sqs-client (tools/random-queue-name)))
+   (create-queue sqs-client (tools/random-queue-name) {}))
   ([sqs-client name]
-   (.createQueue sqs-client name)))
+   (create-queue sqs-client name {}))
+  ([sqs-client name
+    {:keys [fifo
+            visibility-timeout
+            kms-master-key-id
+            kms-data-key-reuse-period]
```

Comment:
No, they aren't used yet because this requires some setup on the AWS side and I haven't checked if/how this can be done using Localstack. I'll have a closer look soon.